### ISSUE TITLE
Allow multiple values for --field options in wp plugin list

### ISSUE THREAD

**Issue by priscillamc** [wp-cli/extension-command#159]
It's not possible to use multiple values in the --field option to customize wp plugin list.

For example, it would be helpful to be able to do the example command below:

``wp plugin list --name=example-plugin1,example-plugin-2``

This should only show the info for the plugins in the list, not all of the plugins.

**Comment by schlessera** [comment]
Correct, this is not possible. The main issue with a mechanism like that is the actual "user interface". If you use a comma for separating values, for example, you cannot have a comma in a value you're looking for anymore.

I wonder if the comma is actually useful in any of the fields here...?

**Comment by Sidsector9** [comment]
@schlessera how about limiting the `--name` argument to accept only plugin slugs? If we do that, we don't have to worry about commas. Maybe instead of --name, we can use --slug for semantic reasons.

**Comment by danielbachhuber** [comment]
Related https://github.com/wp-cli/extension-command/issues/224

**Comment by swissspidy** [comment]
Related:

* https://github.com/wp-cli/wp-cli/pull/6198
